xapian-core/weight/tfidfweight.cc

```diff
@@ -149,6 +149,10 @@ TfIdfWeight::get_idfn(Xapian::doccount termfreq, char c) const
 	    // All documents are indexed by the term
 	    if (N == termfreq) return 0;
 	    return log((N - termfreq) / termfreq);
+    case 'f':
+        return (1.0/termfreq);
+    case 's':
+        return pow(log(N/termfreq),2.0);
```

Comment:
@jaylett I changed my editor tab space configuration to 8 which was previously 4 by default. I've fixed the alignment. Please have a look. Thanks. :)
BTW I'm using Atom code editor. What do you recommend for working with the Xapian code base?